### PR TITLE
CID-15771

### PR DESCRIPTION
**Pull request by ramuned-NHSD**
CID-15771 - duplicates help centre page

### CODE REVIEW

File: src/morethanoneaccount.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+layout: layouts/article.njk
+---
+
+## There is already an NHS login account linked to the NHS number
+
+An NHS login already exists with your NHS number but uses a different email address to the one you entered.
+
+We cannot create this account as you can only have one NHS login connected to your NHS number.
+
+## What can you do
```
Comment (vickicole) [inline src/morethanoneaccount.md:16]: What you can do

Comment (ramuned-NHSD) [reply src/morethanoneaccount.md:16]: image

File: src/morethanoneaccount.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Select 'Login Details' from the 'Manage my Account' page
+2. Select 'Delete my NHS account'
+   This will only delete your online NHS account and will not affect your medical records.
+
+## If you have forgotten your password
+
+Navigate to a service where you need to sign in with your NHS login.
+
+1. Select 'If you cannot remember your password, you can reset it'
```
Comment (vickicole) [inline src/morethanoneaccount.md:32]: Prototype and ticket have "reset it" as a link?

File: src/morethanoneaccount.md (hunk 1 above)
Comment (ramuned-NHSD) [reply src/morethanoneaccount.md:16]: This is what design came up with so I copy pasted the wording from prototype

File: src/morethanoneaccount.md (hunk 2 above)
Comment (vickicole) [reply src/morethanoneaccount.md:32]: Oh sorry just saw the ticket says it's been removed 👍

Comment (ramuned-NHSD) [reply src/morethanoneaccount.md:32]: That is no longer a link as per instructions from designers. I had updated the ticket, prototype is out of date as Kirsten is off sick

File: src/morethanoneaccount.md (hunk 1 above)
Comment (vickicole) [reply src/morethanoneaccount.md:16]: Ticket says it the other way round. Will approve PR now if we are going with prototype

**Review by vickicole — APPROVED** [review]

**Review by PoppleJIW — APPROVED** [review]

**Review by vickicole — APPROVED** [review]

**Review by jacobcallear — APPROVED** [review]

File: src/morethanoneaccount.md (hunk 1 above)
Comment (ramuned-NHSD) [reply src/morethanoneaccount.md]: Has been addressed with R&D and changed :)

**Pull request merged**